lib/translator.js

```diff
@@ -11,9 +11,11 @@ var TranslatedFile = require('./translated-file');
 var PluginError = gutil.PluginError;
 
 
+
 function Translator(options) {
   this.options = lodash.defaults(options, {
       localeDir: 'locale',
+      msgDirs: ['locale'],
```

Comment:
i saw `localeDir` as a sort of canonical dir of languages, but I guess we can modify the `getLocales` to compile that list by iterating over all the locale directories and filtering for uniqueness.